src/pages/home/sidebar/AllSettingsScreen.js

```diff
@@ -0,0 +1,124 @@
+        () => ({
+            sectionStyle: styles.accountSettingsSectionContainer,
+            sectionTranslationKey: 'initialSettingsPage.account',
+            items: [
+                {
+                    translationKey: 'common.workspaces',
+                    icon: Expensicons.Building,
+                    routeName: ROUTES.SETTINGS_WORKSPACES,
+                },
+                {
+                    translationKey: 'allSettingsScreen.subscriptions',
+                    icon: Expensicons.MoneyBag,
+                    action: () => {
+                        Link.openOldDotLink(CONST.ADMIN_POLICIES_URL);
+                    },
+                    shouldShowRightIcon: true,
+                    iconRight: Expensicons.NewWindow,
+                    link: CONST.ADMIN_POLICIES_URL,
+                },
+                {
+                    translationKey: 'allSettingsScreen.cardsAndDomains',
+                    icon: Expensicons.CardsAndDomains,
+                    action: () => {
+                        Link.openOldDotLink(CONST.ADMIN_DOMAINS_URL);
+                    },
+                    shouldShowRightIcon: true,
+                    iconRight: Expensicons.NewWindow,
+                    link: CONST.ADMIN_DOMAINS_URL,
+                },
+            ],
+        }),
+        [styles.accountSettingsSectionContainer],
+    );
+
+    /**
+     * Retuns JSX.Element with menu items
+     * @param {Object} data list with menu items data
+     * @returns {JSX.Element} the menu items for passed data
+     */
+    const getMenuItemsSection = useCallback(
+        (data) => (
+            <View style={[styles.pb4, styles.mh3]}>
+                {_.map(data.items, (item, index) => {
+                    const keyTitle = item.translationKey ? translate(item.translationKey) : item.title;
+
+                    return (
+                        <MenuItem
+                            key={`${keyTitle}_${index}`}
+                            wrapperStyle={styles.sectionMenuItem}
+                            title={keyTitle}
+                            icon={item.icon}
+                            shouldShowRightIcon={item.shouldShowRightIcon}
+                            iconRight={item.iconRight}
+                            disabled={isExecuting}
+                            onPress={singleExecution(() => {
+                                if (item.action) {
+                                    item.action();
+                                } else {
+                                    waitForNavigate(() => {
+                                        Navigation.navigate(item.routeName);
+                                    })();
+                                }
+                            })}
+                            shouldBlockSelection={Boolean(item.link)}
+                            focused={activeRoute && item.routeName && activeRoute.toLowerCase().replaceAll('_', '') === item.routeName.replaceAll('/', '')}
```

Comment:
There is also an inconsistency with the use of activeRoute. In `InitialSettingsPage` we use route names where in `WorkspaceInitialPage` we use screen names. I think we are interested in the screen names since the useActiveRoute returns the screen name.

Let's fix this inconsistency and clear the logic here to be just:
```js
focused={item.routeName && item.routeName === activeRoute}
```